src/components/CheckoutDetailsForm/CheckoutDetailsForm.tsx

```diff
@@ -25,25 +26,44 @@ export const CheckoutDetailsForm = (): ReactElement => {
       onSubmit={(e) => e.preventDefault()}
     >
       <div className="h-5/6 w-full grow">
-        <h1 className="mb-3 text-[20px] font-bold text-[#333F4C]">Order</h1>
+        <div>
+          <h1 className="mb-4 text-[20px] font-[600] text-[#333F4C]">Confirm Order</h1>
+          <hr className="px-1"></hr>
+        </div>
         {lineItemConditional && <OrderDetails items={localOrder.lineItems} />}
-        <p
+        <div
           className={cn(
             lineItemConditional ? 'opacity-0' : 'opacity-100',
-            'relative top-2/4 transform text-center text-black transition-all duration-300'
+            'relative translate-y-[50%] transform items-center justify-center text-center text-[14px] font-[400] text-black transition-all duration-300'
           )}
         >
-          Your order is empty
-        </p>
-      </div>
-      <div className={cn(lineItemConditional ? 'opacity-100' : 'opacity-0', 'mb-3 w-full border-t pt-5 text-[22px] transition-all duration-300')}>
-        <div className="flex w-full justify-between text-[10px] text-[#5A6675]">
-          <span className="">Service Charge:</span>
-          <span className="">{GBPFormat.format(serviceCharge)}</span>
+          <img src={CheckoutBurgerImg} className="mx-auto mb-[24px] h-[50px] w-[80px]"></img>
+          <p>
+            Whoa, you&apos;ve not got <br></br> anything in your order yet
+          </p>
+          <Button
+            colour="yellow"
+            size="small"
+            onClick={navigateToMenu()}
```

Comment:
Could this just be `onClick={navigateToMenu}` ?

Reply:
You might have to change `navigateToMenu` from a function to an arrow function, can't remember if it works otherwise. Just give it a try and resolve this comment if not 🤷